netbox/dcim/tests/test_api.py

```diff
@@ -1828,6 +1839,21 @@ def test_list_devices_brief(self):
             ['display_name', 'id', 'name', 'url']
         )
 
+    def test_list_device_null_region(self):
+
+        Device.objects.create(
+            device_type=self.devicetype1,
+            device_role=self.devicerole1,
+            name='Test Device Null Region',
+            site=self.site2,
+            cluster=self.cluster1
+        )
+
+        url = reverse('dcim-api:device-list')
+        response = self.client.get('{}?region=null'.format(url), **self.header)
+
```

Comment:
Could we also add a test to make sure if region is not defined or is defined but empty it returns all?  Same with multiple values (region=null,12)

Reply:
Spoke with @kobayashi I will enhance the test suite around regions after this PR is merged.